infra/github-runner/prod/container_app_job.tf

```diff
@@ -0,0 +1,21 @@
+module "container_app_job" {
+  source = "github.com/pagopa/dx//infra/modules/github_selfhosted_runner_on_container_app_jobs?ref=main"
+
+  prefix    = local.prefix
+  env_short = local.env_short
+
+  key_vault = {
+    resource_group_name = data.azurerm_key_vault.kv.resource_group_name
+    name                = data.azurerm_key_vault.kv.name
+  }
+
+  container_app_environment = {
+    name                = azurerm_container_app_environment.github_runner.name
+    resource_group_name = azurerm_container_app_environment.github_runner.resource_group_name
+  }
+
+  repo_name              = "trial-system"
```

Comment:
**question**: is it possible to get repo name from terraform?

We defined repo name in another module (https://github.com/pagopa/trial-system/blob/59d0a1e462bf6dc8aee76ee5526d5cf4a4a73c77/infra/repository/github_repository.tf#L2); is it possible to reference it?

Reply:
Probably navigating the state file of the other config.. thanks for feedback, we can think of some easy way to make them work together